Wikipedia/Code/WMFArticleImageProtocol.m

```diff
@@ -23,6 +23,22 @@
 
 NSString* const WMFArticleImageSectionImageRetrievedNotification = @"WMFSectionImageRetrieved";
 static NSString* const WMFArticleImageProtocolHost               = @"upload.wikimedia.org";
+static NSString* const WMFArticleImageProtocolResponseAndDataErrorDomain = @"WMFArticleImageProtocolResponseAndDataErrorDomain";
+
+@interface WMFArticleImageProtocolResponseAndData: NSObject
```

Comment:
Containment for the parts we want to hand off to a background thread for processing.